scripts/scil_tractogram_project_streamlines_to_map.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env python3
```

Comment:
In the name "tractogram_project_streamlines_to_map", there is some redundancy. I would suggest either "tractogram_project_to_map" or "streamlines_project_to_map", but both make for an ugly (imo) name.